awx/ui_next/src/screens/Inventory/InventoryGroups/InventoryGroupAdd/InventoryGroupAdd.test.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { GroupsAPI } from '@api';
+import { createMemoryHistory } from 'history';
+import { act } from 'react-dom/test-utils';
+import { mountWithContexts, waitForElement } from '@testUtils/enzymeHelpers';
+
+import InventoryGroupAdd from './InventoryGroupAdd';
+
+jest.mock('@api');
+
+describe('<InventoryGroupAdd />', () => {
+  let wrapper;
+  let history;
+  beforeEach(async () => {
+    history = createMemoryHistory({
+      initialEntries: ['/inventories/1/groups'],
+    });
+    await act(async () => {
+      wrapper = mountWithContexts(
+        <InventoryGroupAdd
+          setBreadcrumb={() => {}}
+          inventory={{ inventory: { id: 1 } }}
+        />,
+        {
+          context: {
+            router: {
+              history,
+            },
+          },
+        }
+      );
+    });
+  });
+  afterEach(() => {
+    wrapper.unmount();
+  });
+  test('InventoryGroupEdit renders successfully', () => {
+    expect(wrapper.length).toBe(1);
+  });
+  test('cancel should navigate user to Inventory Groups List', async () => {
+    await act(async () => {
+      waitForElement(wrapper, 'isLoading', el => el.length === 0);
+    });
+    expect(history.location.pathname).toEqual('/inventories/1/groups');
```

Comment:
How is this testing cancel navigation

Reply:
When the user cancels the form is should send them to `/inventories/inventory/1/group` which would be the inventory groups list.  I do see that the url in the code is wrong based on what I just typed so I'll fix that.